charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml

```diff
@@ -141,21 +141,85 @@ data:
                 - name: OT_ENABLED
                   value: {{ .Values.application.ray.openTelemetryCollector.enabled | quote }}
 {{- end }}
-            - image: fluent/fluent-bit:1.9.10
-              name: ray-head-logs
+            - image: {{ .Values.application.ray.proxyImage }}
+              securityContext:
+                runAsGroup: 123
+                privileged: true
+                capabilities:
+                  add: ["NET_ADMIN"]
+              imagePullPolicy: Never
```

Comment:
Shouldn't it be `IfNotPresented` or something liek that? :)

Reply:
You are right.  It's for my development environment.  I'll change it `IfNotPresented`.